v-next/hardhat-node-test-reporter/integration-tests/fixture-tests/only-test-with-the-flag/test.ts

```diff
@@ -0,0 +1,21 @@
+import assert from "node:assert/strict";
```

Comment:
I think adding another file to this test case would be useful. Something like:

`ignored-tests.ts`:
```ts
import { describe, it, test } from "node:test";

describe("test excluded because they don't have a .only", () => {
   it("ignored", () => {});
});
```

The reason I think it's worthwhile is I believe that right now we'd display it as a passed test, which we should avoid in a future pr.

Reply:
That's a great point! I added test cases that should be ignored when run with the flag to the fixture that is generated without the flag, but forgot to add it here 🤦 I added it, now, in https://github.com/NomicFoundation/hardhat/pull/5692/commits/fb775b7988cc083212bd55aa38dff95f85f08e28